worker/task/process_event_delivery.go

```diff
@@ -152,7 +152,12 @@ func ProcessEventDelivery(appRepo datastore.ApplicationRepository, eventDelivery
 		attemptStatus := false
 		start := time.Now()
 
-		resp, err := dispatch.SendRequest(e.TargetURL, string(convoy.HttpPost), sig.Payload, g, header, int64(cfg.MaxResponseSize), ed.Headers)
+		maxResponseSize := g.Config.MaxPayloadReadSize
```

Comment:
@danvixent This shouldn't change at all. We only want to limit our ingest size.

Reply:
alright